config/config.example.toml

```diff
@@ -380,7 +380,8 @@ pay_later.klarna = { connector_list = "adyen" }     # Mandate supported payment
 bank_debit.ach = { connector_list = "gocardless" }  # Mandate supported payment method type and connector for bank_debit
 bank_debit.becs = { connector_list = "gocardless" } # Mandate supported payment method type and connector for bank_debit
 bank_debit.sepa = { connector_list = "gocardless" } # Mandate supported payment method type and connector for bank_debit
-bank_redirect.ideal = {connector_list = "stripe,adyen"} # Mandate supported payment method type and connector for bank_redirect
+bank_redirect.ideal = {connector_list = "stripe,adyen,globalpay"} # Mandate supported payment method type and connector for bank_redirect
```

Comment:
why are ideal changes in sofort pr?

Reply:
Globalpay support mandates for ideal. But it was missed in the ideal PR. Hence, this PR also contains minor config changes of `ideal`